cmake/Modules/FindTinyXML2.cmake

```diff
@@ -13,6 +13,40 @@ if(TinyXML2_FOUND)
       set(TINYXML2_LIBRARY tinyxml2)
     elseif(TARGET tinyxml2::tinyxml2)
       set(TINYXML2_LIBRARY tinyxml2::tinyxml2)
+    elseif(TinyXML2_FIND_REQUIRED)
+      message(FATAL_ERROR "Unable to determine target for TinyXML2")
+    endif()
+    list(APPEND TinyXML2_TARGETS ${TINYXML2_LIBRARY})
+  else()
+    # Only perform that logic once
+    if(NOT TARGET tinyxml2::tinyxml2)
```

Comment:
Nitpick: this line could be combined with the previous line into an `elseif` to safe one indentation level.